services/visual-studio-app-center/visual-studio-app-center-releases-osversion.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+
+const schema = Joi.object({
+  app_os: Joi.string().required(),
+  min_os: Joi.string().required(),
+}).required()
+
+module.exports = class VisualStudioAppCenterReleasesOSVersion extends BaseVisualStudioAppCenterService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'visual-studio-app-center/releases/osver',
+      pattern: ':owner/:app/:token',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Visual Studio App Center (Minimum) OS Version',
+        namedParams: {
+          owner: 'jct',
+          app: 'my-amazing-app',
+          token: 'ac70cv...',
+        },
+        staticPreview: this.render({ min_os: '4.1', app_os: 'Android' }),
+        keywords,
+        documentation,
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'min version',
+      color: 'blue',
+    }
+  }
+
+  static render({ app_os, min_os }) {
+    return {
+      message: `${app_os} ${min_os}`,
+    }
```

Comment:
What would you think about using the app os in the label here? For example something like:

![](https://img.shields.io/badge/Android-^4.1-blue)

I'm not sure if that makes sense in this context, but I think that may be a bit more aligned with our other platform/version support badges.

Reply:
I'm not entirely sure how to interpret the confused emoji in the context of the suggestion 😄 

Given the `min_os` naming convention, this badge actually sounds like a better fit for the `platform-support` category instead of `version`, but with both `platform-support` and `version` badges, the badge label is representative of the platform/service. 

What we currently have here: ![](https://img.shields.io/badge/min%20version-Android%204.1-blue) doesn't really fit into that mold, so the label/message need to be tweaked a bit IMO to be consistent with Shields conventions. 

The ones I listed in my last comment were just some suggested alternatives.

https://shields.io/category/platform-support
![image](https://user-images.githubusercontent.com/13042488/66968591-78b74180-f04b-11e9-92a1-14b3b7efb849.png)

https://shields.io/category/version
![image](https://user-images.githubusercontent.com/13042488/66968621-b025ee00-f04b-11e9-8e1b-e017873ad531.png)

Reply:
A couple other options:

- **android | >=4.1**
- **android | 4.1+**

Reply:
I'd like to go with ![](https://img.shields.io/badge/android-4.1%2B-blue) if you guys don't mind. I personally think that it would be the least confusing.

Reply:
SGTM!